src/quo2/components/inputs/title_input/style.cljs

```diff
@@ -30,7 +30,7 @@
   [customization-color blur? override-theme]
   (if blur?
     (colors/theme-colors colors/neutral-100 colors/white override-theme)
-    (colors/custom-color customization-color (if (or (= :dark override-theme) colors/dark?) 60 50))))
+    (colors/custom-color customization-color (if (or (= :dark override-theme) (colors/dark?)) 60 50))))
```

Comment:
maybe custom `color-by-theme` (in foundations/colors) should also include an override theme param?